extensions/ql-vscode/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Fix bug where the `CodeQL: Compare Query` command did not work for comparing quick-eval queries. [#2422](https://github.com/github/vscode-codeql/pull/2422)
 - Update text of copy and export buttons in variant analysis results view to clarify that they only copy/export the selected/filtered results. [#2427](https://github.com/github/vscode-codeql/pull/2427)
 - Add warning when using unsupported CodeQL CLI version. [#2428](https://github.com/github/vscode-codeql/pull/2428)
+- Add `CodeQL: Quick Evaluation Count` command to generate the count of the selected item without having to compute locations or strings.
```

Comment:
Optional: do you think it would be better to include a reason why someone would want to use htis instead of standard quick eval (ie- to save computation time)?

Reply:
Perhaps also some wording like "statistics" or "the number of results produced by the selected logic" so it's more obvious what the count is?